SolrNet/Commands/Parameters/RequestHandlerParameters.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace SolrNet.Commands.Parameters
+{
+    /// <summary>
+    /// Request handler parameters
+    /// </summary>
+    public class RequestHandlerParameters
+    {
+        private readonly string handlerUrl;
+
+        /// <summary>
+        /// URL for query request handler.
+        /// </summary>
+        public string HandlerUrl
```

Comment:
FYI this could also be changed already to:

``` csharp
public string HandlerUrl {get;}

public RequestHandlerParameters(string url)
{
    if (string.IsNullOrEmpty(url))               
        throw new ArgumentException("url");
     this.HandlerUrl = url;
}
```

With other words, no need anymore for private field (not sure which C# version introduced this new way, I think C# 6).